assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -32,12 +32,14 @@ import {
   Icons,
   THEME_CONSTANTS,
   Headline,
+  themeHelpers,
 } from '../../../../../design-system';
-import { KEYBOARD_USER_SELECTOR } from '../../../../utils/keyboardOnlyOutline';
 import DragHandle from './handle';
 
-// If the header is collapsed, we're leaving 8px less padding to apply that from the content.
-const Header = styled.h2`
+// If the header is collapsed, we're leaving 4px less padding to apply that from the content.
+const Header = styled(Headline).attrs({
+  as: 'h2',
+})`
```

Comment:
eyes here from others who have updated this file recently, please! Did I muck up this rebase?

Reply:
This looks good! I reverted my changes. The changes that are in main are the ones that we should follow, and I like what you did here to update the `h2` to `Headline` and the `Headline` to `span` (lower)

Reply:
Ping @miina.

Reply:
@BrittanyIRL The only thing I'm seeing is that now most of the panel seems to be a button and have  a focus line around it, however, on Figma it shows differently, only the chevron and the text should be clickable based on that (unless there's new information I've missed):
PR:
![Screenshot 2021-03-22 at 09 37 01](https://user-images.githubusercontent.com/3294597/111955659-1426ce80-8aea-11eb-916a-cdb6f8933c3d.png)

Figma:
![Screenshot 2021-03-22 at 09 37 58](https://user-images.githubusercontent.com/3294597/111955682-1ab54600-8aea-11eb-850f-922b1b967d9a.png)

Thoughts?
There were also some style changes merged with #6815 -- seems to have caused new conflicts, should be just colour changes, so hopefully something simple.

Reply:
Ah drag, ok i'll take a look. This is kinda tricky because the checklist has the badge that's justified, but perhaps that needs to be reworked so that the text is the only focusable part? 

Thank you!

Reply:
ok, i decided to add a new prop to `Title` through the `SimplePanel` called `hasBadge` just in case other panels have badges in the future that allows the width to expand + `align-items: space-between` allows the focus border to be flexible. 
![Screen Shot 2021-03-26 at 10 40 26 AM](https://user-images.githubusercontent.com/10720454/112671857-fa55f600-8e1f-11eb-96e0-9becbf6e10da.png)
![Screen Shot 2021-03-26 at 10 40 21 AM](https://user-images.githubusercontent.com/10720454/112671858-faee8c80-8e1f-11eb-899d-0d61e186b793.png)